packages/patrol/lib/src/custom_finders/patrol_tester.dart

```diff
@@ -54,7 +64,10 @@ class PatrolTesterConfig {
       existsTimeout: existsTimeout ?? this.existsTimeout,
       visibleTimeout: visibleTimeout ?? this.visibleTimeout,
       settleTimeout: settleTimeout ?? this.settleTimeout,
+      // TODO: remove after andSettle is removed
```

Comment:
Please create an issue for this and link to it from here

for example:

```suggestion
      // TODO: remove after andSettle is removed (see #1234)
```